types/sse.d.ts

```diff
@@ -0,0 +1,90 @@
+export type SSE = {
+    /**
+     * - headers
+     */
+    headers: string;
```

Comment:
here should be SSEHeader type
```suggestion
    headers: SSEHeader;
```